src/test/java/guitests/StartupBoardLauncherTest.java

```diff
@@ -0,0 +1,47 @@
+package guitests;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.loadui.testfx.utils.FXTestUtils;
+import prefs.Preferences;
+import ui.BoardAutoCreator;
+import ui.TestController;
+import ui.UI;
+import ui.issuepanel.PanelControl;
+import static guitests.BoardAutoCreatorTest.verifyBoard;
+
+import static org.junit.Assert.assertEquals;
+import static ui.BoardAutoCreator.SAMPLE_BOARD;
+
+public class StartupBoardLauncherTest extends UITest{
+
+    private PanelControl panelControl;
+    private Preferences testPref;
```

Comment:
It's a private preferences variable called in the test class, hence `testPref`. It's called the same in a number of other test classes like `RepositorySelectorTest`, `MenuControlTest` etc.